core/ecschema-locaters/src/SchemaJsonFileLocater.ts

```diff
@@ -116,3 +116,80 @@ export class SchemaJsonFileLocater extends SchemaFileLocater implements ISchemaL
     return schema as T;
   }
 }
+
+/**
+ * A SchemaLocator implementation for locating JSON Schema files
+ * from the file system using configurable search paths.
+ * @beta This is a workaround the current lack of a full xml parser.
+ */
+export class PublishedSchemaJsonFileLocater extends SchemaJsonFileLocater implements ISchemaLocater {
+  public static defaultSchemaSearchPaths = new Set([
```

Comment:
this seems to be repeated a few times, wonder if we extract this out to a var that is referenced everywhere
what happens today if you update one of these places but not the others?

having a list of these schemas grouped is helpful, but if its not needed you could use a glob to pull in all schemas

Reply:
Done!